west.yml

```diff
@@ -245,7 +245,7 @@ manifest:
       groups:
         - hal
     - name: hal_stm32
-      revision: 5cbc642b1a79d4f373b1587f8c3027f31bf0d30c
+      revision: e51136cb43ae2d7b9742d84a981280a09fb25138
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/pull/90018 will probably be merged before this PR and will update hal_stm32 sha1 to a later version which will include current commit and few others.

To avoid issues, you'll then need to rebase but remove this modification to be sure we don't go back in time in hal_stm32.